src/app/common/SummaryView.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <KSlideout
+    ref="slideOutRef"
```

Comment:
**Suggestion**: This is no longer needed.

```suggestion
```

Reply:
Oh thanks!